File: C#/Урок 043 - continue/Задача 02.cs
class Program
{
    static void Main()
    {
        int min = 1, max = 50;

        System.Console.Write($"Enter a number between {min} and {max}: ");
        int userNumber = System.Convert.ToInt32(System.Console.ReadLine());

        for (int number = min; number <= max; number++)
        {
            if (number == userNumber)
                continue;

            System.Console.Write(number + " ");
        }
    }
}

File: C++/Урок 037 - continue/Задача 02.cpp
#include <iostream>

int main()
{
	const int MIN = 1, MAX = 100;
	int divider = 7;

	for (int number = MIN; number <= MAX; number++)
	{
		if (number % divider != 0)
		{
			continue;
		}
		std::cout << number << "\t";
	}

	return 0;
}